client/i18n/locales/english/translations.json

```diff
@@ -416,6 +416,21 @@
     "help-translate-link": "Help us translate.",
     "project-preview-title": "Here's a preview of what you will build",
     "github-required": "<0>Create a GitHub</0> account if you don't have one. You'll need it when you create the virtual Linux server machine. This process may take a few minutes.",
+    "gitpod": {
+      "intro": "This course runs in a virtual Linux machine using Gitpod. Follow these instructions to start the course:",
+      "step-1": "<0>Create a GitHub</0> account if you don't have one",
+      "step-2": "Click the start button below",
+      "step-3": "Login to Gitpod with your GitHub account if you aren't already",
+      "step-4": "Once the virtual Linux machine is finished loading, start the CodeRoad extension by:",
+      "step-5": "Clicking the \"hamburger\" menu near the top left of the VSCode window,",
+      "step-6": "Going to the \"View\" menu,",
+      "step-7": "Clicking on the \"Command Palette\" option,",
+      "step-8": "and running the \"CodeRoad: Start\" command",
+      "step-9": "Follow the instructions in CodeRoad to complete the course",
+      "continue-project": "Clicking the button below will start a new project. If you have previously started the course, go to <0>your Gitpod dashboard</0> to continue.",
```

Comment:
Same issue as `course` vs `curriculum`. I worry Campers will think there is only **1** Gitpod workspace for the entire curriculum.

So, when they move on to the second project, they will re-read this note, and think to themselves "Oh, well I already have started a Gitpod workspace. I should reopen that one".

Reply:
I added the specific course to the two alerts to make it more clear.